app/views/partials/search_and_filter.scala.html

```diff
@@ -18,37 +18,46 @@
 
 @this(filter_buttons: filter_buttons, textbox: textbox, input_select: input_select)
 
-@(form: Form[_], withFilter: Boolean = true)(implicit request: Request[_], msgs: Messages, appConfig: AppConfig)
+@(
+    form: Form[_],
+    withFilter: Boolean = true,
+    searchLabelKey: String  = "client-filter.text.label",
+    filterLabelKey: String  = "client-filter.select.label",
+    emptyFilterOptionKey: String  = "client-filter.select.label.empty-option"
+)(implicit request: Request[_], msgs: Messages, appConfig: AppConfig)
+
+@searchGridClass = @{if(withFilter) "govuk-grid-column-one-third" else "govuk-grid-column-one-half"}
+@paddingTopType = @{if(withFilter && msgs(searchLabelKey).length > 38) "pad-top-2" else "pad-top-1" }
 
 <div class="govuk-grid-row" role="search" id="search-filter-wrapper">
 
-    <fieldset class="govuk-fieldset">
+    <fieldset class="govuk-fieldset" id="search-and-filter">
```

Comment:
still feels like you shouldn't have a fieldset when it's just the one input - just adds noise for assistive tech